app/src/main/res/layout/dialog_tutorial_upload.xml

```diff
@@ -1,14 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:padding="@dimen/dialog_horizontal_margin">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:padding="@dimen/dialog_horizontal_margin"
+    android:orientation="vertical">
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:drawableStart="@drawable/ic_file_upload_inverted_48dp"
         android:drawablePadding="8dp"
+        app:drawableTint="@color/text"
         android:text="@string/dialog_tutorial_upload"
         android:textAppearance="@android:style/TextAppearance.Theme.Dialog"/>
-
-</RelativeLayout>
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="3dp"
+        />
```

Comment:
Not sure how it looks, to nicely separate the two, you could also use a horizontal line (background color of view to `@color/divider`, layout height to 1dp + padding top and bottom)

Reply:
I deliberately decided to not use line, as for me it is not about two strongly separate things, so strong separation was deliberately not used.